src/components/Header/Navigation/index.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
+
+class HeaderNavigation extends Component {
+  render() {
+    return <React.Fragment>This is the Navigation.</React.Fragment>;
```

Comment:
For this you can just import as `React, { Component, Fragment } from 'react';` and it'll achieve the same thing.